modules/core/src/lib/timeline.js

```diff
@@ -53,30 +44,14 @@ export class Timeline {
   }
 
   setTime(time) {
-    this._setChannelTime(this, time);
+    this.time = Math.max(0, time);
```

Comment:
I wonder if this should throw if `time < 0`. If someone passes a negative time they either (1) don't understand `setTime`'s requirements and would benefit from the message, or (2) didn't expect their code to pass a negative number and would benefit from the message. If throwing is too aggressive, then maybe a console warning would do the job?

Reply:
I don't really consider negative input to be an error. It's just that it will be clamped to 0. If negative values are passed in unintentionally, I'm thinking it should be pretty easy to spot, since the time values won't move (e.g. an animation will stop progressing).